ui/src/components/MotorInput/MotorInput.jsx

```diff
@@ -96,114 +96,76 @@ export default class MotorInput extends React.Component {
     return (
       <div className="motor-input-container">
         <p className="motor-name">{this.props.label}</p>
-        <div className="d-flex">
-          <form className="d-flex" onSubmit={this.handleKey} noValidate>
-            <div style={{ display: 'flex', width: '160px' }}>
```

Comment:
Removing some unnecessary wrappers to keep only the `form`. You may want to hide whitespace changes when reviewing.